Modules/IndividualAssessment/classes/Settings/class.ilIndividualAssessmentSettings.php

```diff
@@ -113,34 +113,58 @@ public function isResultVisible(): bool
     public function toFormInput(
         Field\Factory $input,
         ilLanguage $lng,
-        Refinery $refinery
+        Refinery $refinery,
+        bool $specified_form
     ): \ILIAS\UI\Component\Input\Container\Form\FormInput {
+
+        $title = $input->text($lng->txt("title"))
+                       ->withValue($this->getTitle())
+                       ->withRequired(true);
+        $description = $input->textarea($lng->txt("description"))
+                             ->withValue($this->getDescription());
+        $content = $input->textarea($lng->txt("iass_content"), $lng->txt("iass_content_explanation"))
+                         ->withValue($this->getContent());
+        $record_template = $input->textarea($lng->txt("iass_record_template"), $lng->txt("iass_record_template_explanation"))
+                                 ->withValue($this->getRecordTemplate());
+        $time_place_required = $input->checkbox($lng->txt("iass_event_time_place_required"), $lng->txt("iass_event_time_place_required_info"))
+                                     ->withValue($this->isEventTimePlaceRequired());
+        $file_required = $input->checkbox($lng->txt("iass_file_required"), $lng->txt("iass_file_required_info"))
+                               ->withValue($this->isFileRequired());
+        $file_visible = $input->checkbox($lng->txt("iass_file_visible_examinee"), '')
+                              ->withValue($this->isFileVisible());
+        $notification = $input->checkbox($lng->txt("iass_notify"), $lng->txt("iass_notify_explanation"))
+                              ->withValue($this->isResultVisible());
+
+        $fields = [
+            "title" => $title,
+            "description" => $description,
+            "content" => $content
+        ];
+
+        if (!$specified_form) {
+            $fields['record_template'] = $record_template;
+            $fields['event_time_place_required'] = $time_place_required;
+            $fields['file_required'] = $file_required;
+            $fields['file_visible'] = $file_visible;
+        }
+
+        $fields['result_visible'] = $notification;
```

Comment:
Macht es u.U. Sinn, die Felder in Groups zu verpacken? 
Wir haben mindestens generic, standard/custom und LP bzw workflow.